src/libcharon/sa/ikev2/task_manager_v2.c

```diff
@@ -2011,6 +2011,9 @@ static void trigger_mbb_reauth(private_task_manager_t *this)
 		DBG1(DBG_IKE, "reauthenticating IKE_SA failed");
 	}
 	charon->bus->set_sa(charon->bus, this->ike_sa);
+#ifdef USE_KERNEL_SUPER
+	charon->kernel->write_terminator_tunnel_ike_sa_auth_uptime(charon->kernel, this->ike_sa);
```

Comment:
1. This function is called on make-before-break reauthentication. Need to support break-before-make as well
2. Still this is called on the initiator side. If we're the responder then also need to reset the IKE_SA_AUTH_UPTIME